crypto/fipsmodule/modes/asm/aesv8-gcm-armv8.pl

```diff
@@ -849,31 +849,8 @@
 	eor     $acc_mb, $acc_mb, $rk4v.16b                      // GHASH final-1 block - mid
 	eor     $acc_lb, $acc_lb, $rk3                           // GHASH final-1 block - low
 .Lenc_blocks_less_than_1:                                        // blocks left <= 1
```

Comment:
Do we change these labels so they're not misleading?

Reply:
@nebeid Can you elaborate? I think the number of blocks to be processed is unchanged, or?

Reply:
The label suggests we're processing a partial block, which was the code that was removed, right?
On the other hand, the comment says `blocks left <=1`, so maybe (if I understand correctly)
```suggestion
.Lenc_blocks_eq_1:                                        // blocks left = 1
```

Reply:
Yes, you are right! I'll change it.

Reply:
https://github.com/aws/aws-lc/pull/1639/commits/bef148935a93b4c40c27c6fb5bba480830f0beea